### ISSUE TITLE
[production_histograms] invalid syntax (Trac #1737)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1729]
```text
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.production_histograms.histogram_modules.simulation.rst:15: WARNING: autodoc: failed to import module u'icecube.production_histograms.histogram_modules.simulation.corsika_weight'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/production_histograms/histogram_modules/simulation/corsika_weight.py", line 12
    self.append(Histogram(, , , "FluxSum"))
                          ^
SyntaxError: invalid syntax
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.production_histograms.histogram_modules.simulation.rst:63: WARNING: autodoc: failed to import module u'icecube.production_histograms.histogram_modules.simulation.nugen_weight'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/production_histograms/histogram_modules/simulation/nugen_weight.py", line 12
    self.append(Histogram(, , , "OneWeight"))
                          ^
SyntaxError: invalid syntax

```

Migrated from https://code.icecube.wisc.edu/ticket/1737
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "\n{{{\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.production_histograms.histogram_modules.simulation.rst:15: WARNING: autodoc: failed to import module u'icecube.production_histograms.histogram_modules.simulation.corsika_weight'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/production_histograms/histogram_modules/simulation/corsika_weight.py\", line 12\n    self.append(Histogram(, , , \"FluxSum\"))\n                          ^\nSyntaxError: invalid syntax\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.production_histograms.histogram_modules.simulation.rst:63: WARNING: autodoc: failed to import module u'icecube.production_histograms.histogram_modules.simulation.nugen_weight'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/production_histograms/histogram_modules/simulation/nugen_weight.py\", line 12\n    self.append(Histogram(, , , \"OneWeight\"))\n                          ^\nSyntaxError: invalid syntax\n\n}}}\n", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "cmake", 
    "summary": "[production_histograms] invalid syntax", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2016-06-10T07:44:37", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160610T14:42:40: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"
* olivas commented:

> In changeset 147257:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="147257"
> we'll bring this back at some point. fixes #1729.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas changed milestone from "August 2016" to ""
* olivas commented:

> Milestone August 2016 deleted
